pisa/analysis/analysis.py

```diff
@@ -320,6 +320,40 @@ def get_separate_octant_params(hypo_maker, angle_name, inflection_point) :
 
     return angle_orig, angle_case1, angle_case2
 
+def update_param_values(
```

Comment:
It would be great if this function could also accept a `Pipeline` instance as well as a `DistributionMaker`, to make it usable for simpler cases where only a pipeline is used

Reply:
I added support for passing single Pipeline objects and unit test that it works correctly.